### PR TITLE
Atsss rules

### PR DESCRIPTION
**Pull request by shjohnson-bt**
Expanded on the delivery of ATSSS rules. There are two sets of rules in play - one set that determines which traffic flows are eligible for the ATSSS service, the second set are the actual ATSSS rules that govern how each flow is treated within ATSSS

### CODE REVIEW

**Comment by SpencerDawkins** [comment]
Are both sets of rules called "ATSSS rules" with no qualification in 3GPP docs? 

It would be really convenient if we could distinguish between them in this document. We don't need to confuse readers more :-)

**Review by SpencerDawkins — COMMENTED** [review]
Are both sets of rules called "ATSSS rules" in 3GPP documentation, with no qualifiers?

It would be great if we could distinguish between them in our draft without confusing people more.

File: draft-quic-atsss-reqs.mkd [hunk 1]
```diff
@@ -236,8 +236,8 @@ The Proxy enables the UE to use a transport protocol that supports
 the different access networks even if the server does not support it (i.e., is not multipath-capable
 at transport layer).
 
-The 5G control plane provides the UE and the Proxy with ATSSS rules that specify
-which flows are eligible to the ATSSS service.
+The 5G control plane provides the UE and the Proxy with rules that specify
+which flows are eligible to the ATSSS service (i.e. by mapping them to a Multi-Access PDU Session). Once a Multi-Access PDU Session has been established, a set of ATSSS rules are then delivered to both the UE and the Proxy in order to control the treatment of the flows within the Session.
```
Comment (mirjak) [inline draft-quic-atsss-reqs.mkd]:
```suggestion
which flows are eligible to the ATSSS service (i.e. by mapping them to a Multi-Access PDU Session). Once a Multi-Access PDU Session has been established, a set of ATSSS rules are then delivered to both the UE and the Proxy in order to enable unified treatment of the flows by both the UE and the Proxy within the Session.
```

**Comment by shjohnson-bt** [comment]
Spencer,

I understand on the confusion aspects – I think it is also important that if this doc is reflected back to the 3GPP, we need to show that we have a good grasp of the mechanics of ATSSS.

So the two sets of rules are:

UE Route Selection Policy (URSP)
this is initially delivered to a UE once it first registers on a network (before any data is established) and it instructs the UE how it should be delivering traffic across its available accesses. Each rule will map certain application traffic to a PDU Session and the creation of a PDU Session will include the provision of the IP address (potentially a different one for each PDU Session). For ATSSS, matching traffic flows will be mapped to a Multi-Access PDU Session.
There are also a set of equivalent policy (PCC) rules that are delivered to the UPF which match the URSP for each UE.

ATSSS Rules
There will be a set of ATSSS rules per MA PDU session that specify the ATSSS treatment for each flow e.g. use of MPTCP or ATSSS-LL, smallest delay or priority modes etc

Kind Regards,
Stephen

**Comment by SpencerDawkins** [comment]
This is an improvement - thanks for the back ground (Stephen) and text (Mirja). 

I am thinking the reference to "Multi-Access PDU Session" is a bit more 3GPP-ese than I am hoping for in the document, but we're really going to need to make a document pass looking for terms that we might not need to use - so merging this pull request is fine with me.

**Comment by shjohnson-bt** [comment]
I was trying to avoid too much 3GPP speak and that’s why I didn’t refer to URSP rules. I thought MA-PDU Session might be OK though as we talk about PDU Sessions and Multi-Access Connectivity Services earlier in the document.

S.

File: draft-quic-atsss-reqs.mkd [hunk 2]
```diff
@@ -237,7 +237,7 @@ the different access networks even if the server does not support it (i.e., is n
 at transport layer).
 
 The 5G control plane provides the UE and the Proxy with rules that specify
-which flows are eligible to the ATSSS service. Additionally, a set of ATSSS rules are delivered to the UE and the Proxy that control how each flow is treated within the ATSSS service.
+which flows are eligible to the ATSSS service (i.e. by mapping them to a Multi-Access PDU Session). Additionally, a set of ATSSS rules are delivered to the UE and the Proxy that control the treatment of each of these flows within this MA-PDU Session.
```
Comment (obonaventure) [inline draft-quic-atsss-reqs.mkd]: We probably need to define earlier what is a MA-PDU session.

Comment (mpiraux) [reply draft-quic-atsss-reqs.mkd]: This is now defined.

**Pull request merged**